### PR TITLE
bump lower to v1.9

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -20,7 +20,6 @@ steps:
     matrix:
       setup:
         version:
-          - "1.6"
           - "1"
```
Comment (ranocha) [inline .buildkite/pipeline.yml:23]:
```suggestion
          - "1.9"
          - "1"
```
If we set the lower version to v1.9, shouldn't we run tests specifically on Julia v1.9 (given that v1 may soon be v1.10)?

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -26,7 +26,6 @@ jobs:
           - Multithreading
         version:
           - '1'
```
Comment (ranocha) [inline .github/workflows/CI.yml:28]:
```suggestion
          - '1.9'
          - '1'
```
See above

File: .github/workflows/Downstream.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: [1,1.6]
+        julia-version: [1]
```
Comment (ranocha) [inline .github/workflows/Downstream.yml:17]:
```suggestion
        julia-version: [1.9, 1]
```
See above

**Comment by ChrisRackauckas** [comment]
For now this is just doubling CI, but why not. In the near future v1.10 should just be the new LTS and we'll drop this.

**Pull request merged**